draft-huque-dnsop-multi-alg-rules.xml

```diff
@@ -206,7 +206,8 @@
         anchor set DNSKEY algorithms are supported by a validating resolver.
       </t>
       <t>
-        This document proposes modifications of the signing and validation rules
+        This document proposes modifications to the IANA signing algorithm registry and
+        modifications of the signing and validation rules
```

Comment:
*minor* modifications

Reply:
sure, although I meant the other way around as to not scare away readers. :) The rules are not changing much, as validators will keep doing what they're doing (except when phasing out an algorithm), and multi-signers will also mostly keep doing what they're doing.

Reply:
Ah I assumed your suggestion was on the recently added text.  I've swapped it so its now "modifications to registry and minor modifications to rules".  right?

Reply:
ah, I now understand why you thought that! Thanks.